notifications_utils/logging.py

```diff
@@ -16,46 +16,60 @@
 logger = logging.getLogger(__name__)
 
 
+def build_log_line(extra_fields):
+    fields = []
+    if 'service_id' in extra_fields:
+        fields.append(str(extra_fields.get('service_id')))
+    fields += [extra_fields.get('method'), extra_fields.get('url'), str(extra_fields.get('status'))]
```

Comment:
if any of method, url, or status aren't in, you'll get `TypeError: sequence item 1: expected str instance, NoneType found` when it calls `' '.join(fields)`. otherwise there's no point in calling `.get` instead of `extra_fields['method']`